q2_moshpit/partition/mags.py

```diff
@@ -46,7 +47,7 @@ def partition_feature_data_mags(
         for mag_fp, mag_id in _mag:
             duplicate(mag_fp, result.path / os.path.basename(mag_fp))
 
-        # If num_partitions == num_mags we will only have gone through one
+        # If num_partitions == num_samples we will only have gone through one
```

Comment:
Why `num_samples` and not `num_mags` in this coment? I thought that in `FeatureData[MAG]` artifacts we don't have samples. Besides num_mags is the name of the variable that actually gets compared in the _if_ statement below. 

```suggestion
        # If num_partitions == num_mags we will only have gone through one
```